content/javascript/concepts/arrays/arrays.md

```diff
@@ -57,22 +55,77 @@ Individual elements in the array can be accessed using the array name and the el
 array[index]
 ```
 
-This code accesses the value of the first element in the `cars` array:
+This code accesses the value of the first element in the `fruit` array:
 
 ```js
-var cars = ['Saab', 'Volvo', 'BMW'];
+const fruit = ['Apple', 'Orange', 'Banana'];
 
-console.log(cars[0]);
-// Output: Saab
+console.log(fruit[0]);
+// Output: Apple
 ```
 
 Array elements can be changed by accessing the element and assigning a new value to it.
 
 ```js
-var cars = ['Saab', 'Volvo', 'BMW'];
+const fruit = ['Apple', 'Orange', 'Banana'];
+
+fruit[1] = 'Mango';
+
+console.log(fruit[1]);
+// Output: Mango
+```
+
+Accessing an array using an unused index will return `undefined`. However, a new value can still be assigned to an unused index of an array. When doing so, any gaps in the assigned indices will remain `undefined`.
+
+```js
+const fruit = ['Apple', 'Orange', 'Banana'];
 
-cars[1] = 'Tesla';
+fruit[4] = 'Mango'; // a valid assignment
 
-console.log(cars[1]);
-// Output: Tesla
+console.log(fruit[3]);
+// Output: undefined
+```
+
+## Nested Arrays
+
+Any object can be an element of an array. This includes other arrays. When an array has one or more arrays as elements, it is referred to as a "nested array". Accessing the elements of a nested array is similar to accessing the elements of a regular array. It just requires the additional index to reference the element in the inner array.
+
+```js
+// Create a nested array
+const food = [
+  ['Apple', 'Orange', 'Banana'],
+  ['Strawberry', 'Blueberry', 'Raspberry'],
+  ['Potato', 'Carrot', 'Broccoli'],
+];
+
+// Retrieve the object at index 2 of the array at index 1
+console.log(food[1][2]);
+// Output: Raspberry
+
+// Retrieve the object at index 1 of the array at index 0
+console.log(food[0][1]);
+// Output: Orange
+```
+
+The same process applies to arrays that themselves contain nested arrays. To access the elements in the "bottom" array requires an additional index for each level of nesting.
+
+```js
```

Comment:
Wondering if we could turn this one into a `codebyte/js` example since this is the last snippet of the entry?